src/Modules/Backend/Domain/UserGroup/UserGroup.php

```diff
@@ -89,7 +89,8 @@ public function getId(): int
 
     public static function fromDataTransferObject(UserGroupDataTransferObject $userDataTransferObject): self
     {
-        $userGroup = $userDataTransferObject->hasEntity() ? $userDataTransferObject->getEntity() : new self($userDataTransferObject->name);
+        $userGroup = $userDataTransferObject->hasEntity()
+            ? $userDataTransferObject->getEntity() : new self($userDataTransferObject->name);
```

Comment:
Ik vind dat niet leesbaar. @carakas wat denk jij?

Reply:
Goh, ik gebruik dat vrij vaak dus heb hier geen problemen mee, ik heb dit persoonlijk liever dan if else maar als dat de code leesbaarder maakt in dit geval mag dat gerust aangepast worden voor mij

Reply:
ok